scripts/lb-wrapper

```diff
@@ -2,47 +2,76 @@
 
 LOG_FILE="/var/log/xklb.log"
 XKLB_EXECUTABLE="${XKLB_EXECUTABLE:-lb}"
+VERBOSITY="-vv"
 TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
 SURVEY_DB_FILE="${TMP_DOWNLOADS_DIR}/survey.db"
 URL="$1"
+
+# Or download largest possible HD-style / UltraHD videos, to try to force
+# out-of-memory "502 Bad Gateway" for testing of issues like #37 and #79
+# FORMAT_OPTIONS="--format-sort size"
+
 FORMAT_OPTIONS="--format best --format-sort 'tbr~1000'"
-XKLB_FULL_CMD="${XKLB_EXECUTABLE} tubeadd ${SURVEY_DB_FILE} ${URL} --verbose && ${XKLB_EXECUTABLE} dl ${SURVEY_DB_FILE} --prefix ${TMP_DOWNLOADS_DIR} --write-thumbnail ${FORMAT_OPTIONS} --video ${URL} --verbose"
+XKLB_FULL_CMD="${XKLB_EXECUTABLE} tubeadd ${SURVEY_DB_FILE} ${URL} ${VERBOSITY} \
+&& ${XKLB_EXECUTABLE} dl ${SURVEY_DB_FILE} --prefix ${TMP_DOWNLOADS_DIR} \
+--video ${URL} ${FORMAT_OPTIONS} --write-thumbnail ${VERBOSITY}"
+
 
 mkdir -p ${TMP_DOWNLOADS_DIR}
 
-# Function to log messages (and errors especially!)
+# Function to log messages e.g. with level "Info" or "Error"
 log() {
-    echo "$(date +'%Y-%m-%d %H:%M:%S') - $1" | tee -a ${LOG_FILE}
+    local level=$1
+    local message=$2
+    if [[ $message =~ ^downloading* ]]; then
```

Comment:
What about using `[[ $message == *"downloading"* ]]` for simplicity, as it efficiently checks for the presence of 'downloading' in the string, avoiding the overhead of regex?